src/routes/posts.js

```diff
@@ -33,7 +33,32 @@ const createPost = async (req, res, next) => {
     }
 }
 
+// GET ALL ARTICLES (feed)
+
+const fetchPosts = async (req, res, next) => {
+    try {
+        const feed = await db.query('SELECT * FROM posts')
+        const allArticles = feed.rows
+
+        res.status(201).json({
```

Comment:
so according to HTTP conventions, `201` is for a request that creates a new resource. Since we're only fetching here, `200` will be more appropriate 

more on that here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses

Reply:
i didn't know this, thanks so much